app/services/cocina_generator/structural/file_generator.rb

```diff
@@ -34,7 +34,7 @@ def request_file_attributes
         {
           type: Cocina::Models::ObjectType.file,
           version: work_version.version,
-          label: attached_file.label,
+          label: attached_file.label || '',
```

Comment:
I see at least a couple of `attached_file.label || ''` in this PR. Might it be worth tossing something like this at the `AttachedFile` model (like what we're now doing for paths above)?

```ruby
def label
  super || ''
end
```

❓

Reply:
Better yet, I'll make it the DB default.

Reply:
Good call!

Reply:
Path too?

Reply:
Path needs a value else we get it from the attached file. So I don't think we want to default to ''.